examples/portworx/getting_started/README.md

```diff
@@ -0,0 +1,190 @@
+# Portworx add-on for EKS Blueprint
+
+This  guide helps you install portworx on EKS environment using EKS Blueprints and its kubernetes add-on module. In this guide, we create a custom IAM policy and attach it to the node groups in EKS cluster to provide Portworx the required access.
```

Comment:
Update this to using IRSA

Reply:
@askulkarni2  see that the addon requires creation of an IAM policy as a prereq. This is inconvenient for the users and requires extra steps outside of the blueprint. Do we have an example of inlining node group policies so that those are included automatically?